src/components/internal-header/_internal-header.scss

```diff
@@ -27,14 +27,20 @@
 .hmrc-internal-header__logo-link {
   display: table;
 
-  &::before {
-    content: "";
-    display: table-cell;
-    width: 52px;
+  p {
```

Comment:
Would be good if this wasn't an element selctor but a BEM one. I'd move it out of the containing selector and call it something like `.hmrc-internal-header__logo-text` but keep the name consistent with hmrc-banner if possible.